automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterFixtureSpec.scala

```diff
@@ -10,6 +10,7 @@ import org.broadinstitute.dsde.workbench.util.addJitter
 import org.scalatest.{BeforeAndAfterAll, Outcome, fixture}
 
 import scala.concurrent.duration._
+import scala.language.postfixOps
```

Comment:
I was sick of compiler warnings so added this in a couple places

Reply:
I think you can just add https://github.com/DataBiosphere/leonardo/blob/develop/project/Settings.scala#L72 to automation's setting.scala file

Reply:
Ah cool